workspace.json

```diff
@@ -1496,6 +1496,12 @@
             "jestConfig": "libs/application/templates/driving-license/jest.config.js",
             "passWithNoTests": true
           }
+        },
+        "extract-strings": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node libs/localization/scripts/extract 'libs/application/templates/passport/src/**/*.{js,ts,tsx}'"
```

Comment:
shouldn't this be `'libs/application/templates/driving-license/src/**/*.{js,ts,tsx}'

Reply:
whoops